core-services/src/main/java/org/zalando/nakadi/service/timeline/LocalLockManager.java

```diff
@@ -0,0 +1,84 @@
+package org.zalando.nakadi.service.timeline;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import java.io.Closeable;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.TimeoutException;
+import java.util.function.Consumer;
+
+@Service
+public class LocalLockManager {
```

Comment:
as i understood the class consist methods from TimeSyncImpl

Reply:
yes